### ISSUE TITLE
Arcanic Pulsar issue

### ISSUE THREAD

**Issue by bywbilly** [WeakAuras/WeakAuras2#1589]
**Describe the bug**

I tried to monitor the expiration time of Arcanic Pulsar but I failed since when the stack is 3 the  aura_env.expirationtime will simply be 0.

**Do you have an error log of what happened?**
 

I can show you the output results and the code I modified. 
![image](https://user-images.githubusercontent.com/10766902/63311662-d2f68900-c2cc-11e9-8547-b3e9ef3660e7.png)
The code:
![image](https://user-images.githubusercontent.com/10766902/63311684-e275d200-c2cc-11e9-9fa5-1786611b50a2.png)

**To Reproduce**
 
Basically, use the same WA and on a balanced druid, you can reproduce.
**Screenshots**

See above
**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

No
**Which version of WeakAuras are you using?**

2.13.4
**Was it working in a previous version? If yes, which was the last good one?**

Haven't tried
**Additional context**

``` HERE ```

**Comment by emptyrivers** [comment]
> Basically, use the same WA and on a balanced druid, you can reproduce.

You did not provide said WA. You provided screenshots, which unfortunately cannot be imported into WeakAuras. If you provide an import string then I may be able to glean more information on the issue you are having. However...

The code you provided a screenshot of is accessing userdata in aura_env (that is, data that WeakAuras does not supply or maintain), we make no guarantees on that data. So, if this is the only code in your aura then my answer is that you are simply Doing It Wrong™, and I advise you to come to our discord (link [here](https://discord.gg/wa2)) for more help.

**Comment by emptyrivers** [comment]
No response from reporter.

**Comment by InfusOnWoW** [comment]
That's because arcan pulsar has no duration.

And also what you want to do is easily accomplished via conditions.
